recipes/protobuf/all/test_package/conanfile.py

```diff
@@ -19,6 +20,8 @@ def requirements(self):
     def generate(self):
         tc = CMakeToolchain(self)
         tc.cache_variables["protobuf_LITE"] = self.dependencies[self.tested_reference_str].options.lite
+        if Version(self.dependencies["protobuf"].ref.version) >= "4.25.0" and not self.settings.compiler.get_safe("cppstd", False):
+            tc.variables["CMAKE_CXX_STANDARD"] = "14"
```

Comment:
I realized there's a better way of doing this, since in the test package we're in control of the CMake version, in the CMake we can add the following:
```cmake
if (protobuf_VERSION VERSION_LESS "4.0.0")
    target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
else()
    target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)
endif()
```
In place of the existing `target_compile_features`.

Reply:
I will try and see.